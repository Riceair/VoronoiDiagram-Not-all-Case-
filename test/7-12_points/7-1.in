7
318 167
150 287
445 271
209 451
448 419
298 303
174 161