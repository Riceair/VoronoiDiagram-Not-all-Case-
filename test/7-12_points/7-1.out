P 150 287
P 174 161
P 209 451
P 298 303
P 318 167
P 445 271
P 448 419
E -102883 37446.38 217.48 355.34
E -78943.03 -14843.62 230.27 237.01
E 217.48 355.34 230.27 237.01
E 217.48 355.34 326.62 420.97
E 230.27 237.01 243.44 225.51
E 243.44 225.51 361.91 242.93
E 243.44 225.51 4026 -90556
E 326.62 420.97 384.4 346.26
E 326.62 420.97 20488.5 151005
E 361.91 242.93 384.4 346.26
E 361.91 242.93 65901.5 -79791
E 384.4 346.26 93686.5 -1545
